public/app/features/alerting/unified/components/alert-groups/AlertGroup.tsx

```diff
@@ -35,7 +35,10 @@ export const AlertGroup = ({ alertManagerSourceName, group }: Props) => {
           />
           {Object.keys(group.labels).length ? (
             <Stack direction="row" alignItems="center">
-              <AlertLabels labels={group.labels} size="sm" />
+              <Text element="h2" variant="body">
+                <AlertLabels labels={group.labels} size="sm" />
+              </Text>
```

Comment:
This one feels a little strange to me, having a list of elements inside the H2 that define the heading. 
What do you think about the `Grouped by: ....` being that second level heading?

![image](https://github.com/grafana/grafana/assets/4377076/9611e3a1-1696-4515-9a69-9eec55eb2bb0)

Reply:
I agree it feels strange, but it is better than nothing. That could be an idea. At the top level, is it possible to have multiple of those groups? Because then I'd really want something for quick screen reader navigation, even if they'd be `h3` instead. Alternatively give the lists good labels.